java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/JobInfoManager.scala

```diff
@@ -161,9 +161,13 @@ object JobInfoManager {
     } catch {
       case e: SQLException =>
         e.printStackTrace()
-    } finally if (pstmt != null) try {
-      pstmt.close()
-    } catch {
+    } Using.Manager { use =>
```

Comment:
I'm  afraid it is not the correct way to with `Using.Manager`, here we want to automatically close the `pstmt` object in the try block (that in line 155), so u should replace the try block with `Using.Manager`. In my mind, the second try ... catch block is unnecessary